package.json

```diff
@@ -24,7 +24,7 @@
   },
   "devDependencies": {
     "@gplane/tsconfig": "^6.1.0",
-    "@swc/core": "^1.3.106",
+    "@swc/core": "~1.5.24",
```

Comment:
It's just dev dependencies here, so it's safe.

```suggestion
    "@swc/core": "^1.5.24",
```

Reply:
The reason that I changed '^' to '~' :
Yesterday I pulled the latest code. 
Then I run "npm run test".  and terminal showed error.
Found package.json specified @swc/core@^1.3.106 but I installed the latest version(use bun). which causes the error.

Reply:
That's unrelated. Both `^1.5` and `~1.5` are resolve to 1.5.x.